ding/framework/parallel.py

```diff
@@ -253,24 +253,35 @@ def send_rpc(self, func_name: str, *args, **kwargs) -> None:
             - fn_name (:obj:`str`): Function name.
         """
         if self.is_active:
-            payload = {"f": func_name, "a": args, "k": kwargs}
+            topic = func_name + "::"
```

Comment:
whether more identifiers need to be added for avoiding misuse. (like `topic::`)

Reply:
I only split the first `::` when receiving the message, there is no need to add additional letters.